### ISSUE TITLE
Install fails on 64 bit Strawberry

### ISSUE THREAD

**Issue by plicease** [plicease/Shell-Guess#4]
Actually Win32::Process::Info fails:

```
N:\home\ollisg>cpanm Win32::Process::Info -v
cpanm (App::cpanminus) 1.7009 on perl 5.020001 built for MSWin32-x64-multi-thread
Work directory is C:\Users\ollisg/.cpanm/work/1421845224.26724
You have make N:\lang\perl\strawberry\x64\5.20.1\c\bin\dmake.exe
You have LWP 6.08
Falling back to Archive::Tar 2.02
You have N:\program32\GnuWin32\bin\unzip.exe
Checking if you have ExtUtils::MakeMaker 6.58 ... Yes (6.98)
Checking if you have ExtUtils::Install 1.46 ... Yes (1.68)
Searching Win32::Process::Info on mirror http://mirror.sydney.wdlabs.com/cpan ...
Downloading index file http://mirror.sydney.wdlabs.com/cpan/modules/02packages.details.txt.gz ...
--> Working on Win32::Process::Info
Fetching http://mirror.sydney.wdlabs.com/cpan/authors/id/W/WY/WYANT/Win32-Process-Info-1.021.tar.gz ... OK
Unpacking Win32-Process-Info-1.021.tar.gz
Entering Win32-Process-Info-1.021
Checking configure dependencies from META.json
Checking if you have Module::Build 0.36 ... Yes (0.4210)
Running Build.PL
Configuring Win32-Process-Info-1.021 ... Created MYMETA.yml and MYMETA.json
Creating new 'Build' script for 'Win32-Process-Info' version '1.021'
OK
Checking dependencies from MYMETA.json ...
Checking if you have Carp 0 ... Yes (1.3301)
Checking if you have Test::More 0.88 ... Yes (1.001006)
Checking if you have strict 0 ... Yes (1.08)
Checking if you have File::Spec 0 ... Yes (3.48)
Checking if you have Time::Local 0 ... Yes (1.2300)
Checking if you have warnings 0 ... Yes (1.23)
Building and testing Win32-Process-Info-1.021 ... Building Win32-Process-Info
t\basic.t .. 1/? Win32::API::Call: parameter 3 had a buffer overflow at C:\Users\ollisg\.cpanm\work\1421845224.26724\Win32-Process-Info-1.021\blib\lib/Win32/Process/Info/NT.pm line 382.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 6.
t\basic.t .. Dubious, test returned 255 (wstat 65280, 0xff00)
All 6 subtests passed

Test Summary Report
-------------------
t\basic.t (Wstat: 65280 Tests: 6 Failed: 0)
  Non-zero exit status: 255
  Parse errors: No plan found in TAP output
Files=1, Tests=6,  0 wallclock secs ( 0.02 usr +  0.02 sys =  0.03 CPU)
Result: FAIL
Failed 1/1 test programs. 0/6 subtests failed.
FAIL
! Installing Win32::Process::Info failed. See C:\Users\ollisg\.cpanm\work\1421845224.26724\build.log for details. Retry with --force to force install it.
```

**Comment by plicease** [comment]
can we use something like this:

https://gist.github.com/hasegawayosuke/252992

to implement our own getppid?

**Comment by plicease** [comment]
The code in that gist appears to be for fake C.NET or something.  Grrr.

**Comment by plicease** [comment]
This looks more promising.

**Comment by plicease** [comment]
fixed in 0.06!
